modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java

```diff
@@ -849,10 +890,6 @@ private CPDefinition _updateNestedResources(
 			ProductConfigurationUtil.updateCPDefinitionInventory(
 				_cpDefinitionInventoryService, productConfiguration,
 				cpDefinition.getCPDefinitionId());
-
-			ProductConfigurationUtil.updateCPDAvailabilityEstimate(
```

Comment:
@RenanUmV Why are some of the logic in this class removed? It's very hard to see the reason from the commit message, as it just says "merge xx into xx". Please separate the merge and the removal in different commits, and give detailed explanation.